crates/ty/tests/cli/main.rs

```diff
@@ -808,6 +808,32 @@ fn can_handle_large_binop_expressions() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test_case::test_case("concise")]
+#[test_case::test_case("full")]
+#[test_case::test_case("github")]
+#[test_case::test_case("gitlab")]
```

Comment:
Nit: I would probably create a helper function and call that from separate `concise_output`, `full_output` etc. functions over using test case, given that the test cases are very limited (has better IDE support)